src/pages/Following/Following.jsx

```diff
@@ -1,38 +1,84 @@
-import { useState } from 'react';
-import { FollowingTopSpan, FollowingMain, FollowingMainUl } from './styled';
+import { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import {
+  FollowingTopSpan,
+  FollowingMain,
+  FollowingMainUl as FollowingUserItem,
+} from './styled';
 
 import FollowingBtn from '../../components/atoms/Button/Button';
 
 import TopBasicNav from '../../components/CommonUI/Nav/TopBasicNav/TopBasicNav';
 import UserItem from '../../components/CommonUI/UserItem/UserItem';
 import TabMenu from '../../components/CommonUI/TabMenu/TabMenu';
+import getFollowingListApi from '../../api/profile/getfollowing';
+
+import { accountNameValue } from '../../atoms';
+import { follow, unfollow } from '../../api/profile/follow';
+import getProfile from '../../api/profile/getmyprofile';
 
 const Following = () => {
-  const [tit, setTit] = useState(true);
-  const [color, setColor] = useState(true);
+  const myAccountName = useRecoilValue(accountNameValue);
```

Comment:
```suggestion
  const accountName = useRecoilValue(accountNameValue);
```
accountName이 따로 써지지 않고, api 함수 호출 시에도 accountName을 인자로 주셨으면 맞춰주시는 게 보기 좋을 것 같습니다! 그리고 혹시 여기서 localStorage 대신 recoilValue를 사용하신 이유가 있을까요??

Reply:
수정완료 했습니다!
특별한 이유는 없고 그냥 저희 프로젝트에 리코일이 쓰여서요 ㅎㅎ..